modules/InfiniteStories/InfiniteStories.module.scss

```diff
@@ -10,4 +10,14 @@
     display: flex;
     margin: $spacing-8 auto 0;
     color: $color-base-white;
+    transition: background-color 0.15s ease-in;
+
+    &:hover {
+        background-color: var(--prezly-accent-color-tint) !important;
+    }
+
+    &:active,
+    &:focus {
+        background-color: var(--prezly-accent-color-shade) !important;
+    }
```

Comment:
This is strange. I assumed that the Load More link and Subscribe link should both be primary buttons that share the  same styles. Am I wrong?
This is how the Subscribe buttons looks like now. It looks the same when idle, but different when hovered (and has no active styles)

https://user-images.githubusercontent.com/1641218/185448959-5ad30791-017d-4630-bd85-3ce5cff2bc38.mov

Reply:
I agree, we definitely need solidified components for basic stuff.

And we also need a designer to look at this ticket once again :)

Reply:
If Al approves, then yes :)